test/evergreen/print_python_stack_trace.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
```

Comment:
Suggest using `python3` if there's no plan to support python2.

Reply:
Updated in [d6b8174](https://github.com/wiredtiger/wiredtiger/pull/7645/commits/d6b817445834ae25c1f1a883cb5ee7d2d3860373)